test/libyul/yulSyntaxTests/invalid/unicode_string_direction_override.sol

```diff
@@ -0,0 +1,6 @@
+{
+    // pop 1
+    let s := unicode"underflow ‬";
+}
+// ----
+// ParserError 1856: (35-47): Literal or identifier expected.
```

Comment:
Is this the right error?

Reply:
Looks like `AsmParser` doesn't accept unicode directional sequences.
It fails as `currentToken()` returns 'Token::Illegal'. So, I would say it is fine for now, so it is not even possible to get this problem in yul, but we would need to decide how to proceed on this in yul...

I created issue so we can discuss this topic #10622